Testing/SystemTests/scripts/InstallerTests.py

```diff
@@ -76,8 +78,9 @@
 log("Running system tests. Log files are: '%s' and '%s'" % (testRunLogPath,testRunErrPath))
 try:
     run_test_cmd = '%s %s %s/runSystemTests.py --loglevel=%s --executable="%s" --exec-args="%s"' % \
-                (installer.python_cmd,  installer.python_args,
-                 THIS_MODULE_DIR, options.log_level, installer.python_cmd, installer.python_args)
+                (installer.python_cmd, installer.python_args, THIS_MODULE_DIR,
+                options.log_level, installer.python_cmd, installer.python_args)
+    run_test_cmd += " -j=%i --quiet --output-on-failure" % options.ncores
```

Comment:
The `-j=` part here doesn't seem to work as we would like. The PR test log shows: 

```
python /home/builder/jenkins-linode/workspace/pull_requests-rhel7/Testing/SystemTests/scripts/InstallerTests.py -o -d /home/builder/jenkins-linode/workspace/pull_requests-rhel7/build -j 13 --exclude-in-pull-requests
Usage: runSystemTests.py [options]
 
runSystemTests.py: error: option -j: invalid integer value: '=13'
```